app/src/androidTest/java/com/swent/assos/AssoDetailsTest.kt

```diff
@@ -36,6 +37,7 @@ class AssoDetailsTest : SuperTest() {
             assert(DataCache.currentUser.value.following.contains(assoId))
           }
         }
+        sleep(1000)
```

Comment:
use composeTestRule.waitForIdle() to wait for navigation instead of force waiting